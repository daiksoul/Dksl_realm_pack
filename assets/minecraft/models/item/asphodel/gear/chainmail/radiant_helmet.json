{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/chainmail_helmet",
		"layer1": "misc/radiant_symbol"
	}
}
