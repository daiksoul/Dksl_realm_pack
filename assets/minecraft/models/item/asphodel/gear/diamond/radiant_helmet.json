{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/diamond_helmet",
		"layer1": "misc/radiant_symbol"
	}
}
