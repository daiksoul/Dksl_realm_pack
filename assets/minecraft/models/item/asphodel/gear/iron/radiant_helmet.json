{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/iron_helmet",
		"layer1": "misc/radiant_symbol"
	}
}
