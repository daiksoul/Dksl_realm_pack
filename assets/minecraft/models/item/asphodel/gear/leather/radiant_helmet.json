{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/leather_helmet",
		"layer1": "item/leather_helmet_overlay",
		"layer2": "misc/radiant_symbol"
	}
}
