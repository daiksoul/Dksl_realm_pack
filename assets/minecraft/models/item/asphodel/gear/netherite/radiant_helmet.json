{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/netherite_helmet",
		"layer1": "misc/radiant_symbol"
	}
}
