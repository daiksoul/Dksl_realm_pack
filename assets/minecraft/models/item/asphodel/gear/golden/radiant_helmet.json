{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/golden_helmet",
		"layer1": "misc/radiant_symbol"
	}
}
